### DISCUSSION TITLE
How does the AVIF quality option map to avifenc?

### DISCUSSION THREAD

**Discussion by svivian** [ImageMagick/ImageMagick#5371] [Digital Image Processing]
I had been using `avifenc` until I realised that ImageMagick can do AVIF now. But IM has different quality settings: 0-100 with higher being better, while avifenc is 0-63 with lower being better. Is there a way to convert one to the other? For example with this:

    avifenc --min 16 --max 16 input.png output.avif

I tried using 80 in IM and it came out around the same file size but the quality wasn't quite the same.

**Comment by mgenware** [comment]
From IM source code, it passes quality value to libheif, which maps 1-100 range to internal encoder range, e.g. 0-63 for AVIF

https://github.com/ImageMagick/ImageMagick/blob/1bca2e1a28bbc36143d4d39d7ead62928be04db1/coders/heic.c#L1303C1-L1303C1

https://github.com/strukturag/libheif/issues/729
